bin/speed-tweet

```diff
@@ -22,4 +22,4 @@ if(argv.help){
 }
 
 var recurTest = require('../lib/recur-test')
-recurTest(argv.t, argv.r)
+recurTest(argv.r, argv.t, argv.i)
```

Comment:
I know.. but for some reason that yargs is not taking the values correctly from command line. Maybe updating the library...